include/ConfigManager.h

```diff
@@ -230,8 +235,27 @@ class EXPORT ConfigManager
 						const QString & value );
 	void deleteValue( const QString & cls, const QString & attribute);
 
-	void loadConfigFile();
-	void saveConfigFile();
+	struct ConfigLoadResult
+	{
+		ConfigLoadResult() :
+			failuresOccurred(false) {}
+
+		ConfigLoadResult(int _errorLine, int _errorColumn,
+				 QString const & _errorString) :
+			failuresOccurred(true),
+			errorLine(_errorLine),
+			errorColumn(_errorColumn),
+			errorString(_errorString)
+		{}
+
+		bool failuresOccurred;
+		int errorLine;
+		int errorColumn;
+		QString errorString;
```

Comment:
@Wallacoloo That was exactly my reasoning. I consider member variables prefixed with `m_` private to the class and only accessible by getters and setters (if at all). I did not want to bloat the rather simple class `ConfigLoadResult` with setters and getters and calling something like `configLoadResult.m_failuresOccured` just seemed wrong. :)